tests/tpu/test_lora.py

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: Apache-2.0
+import vllm
+from vllm.lora.request import LoRARequest
+
+
+def test_lora_hotswapping():
```

Comment:
Is the test trying to verify if vLLM can swap a LoRA model from disk to memory and vice versa?

Reply:
Nope, it makes sure we can swap adapters between host and TPU memory